contrib/openpgp-card-tools/template.py

```diff
@@ -0,0 +1,17 @@
+pkgname = "openpgp-card-tools"
+pkgver = "0.10.0"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo", "pkgconf"]
+makedepends = ["rust-std", "pcsc-lite-devel", "nettle-devel"]
```

Comment:
can add bzip2-devel too i think

Reply:
(since i added the .pc file just now that works for bzip2-sys by default unless something is passing the static feature)

Reply:
would be great if bzip2 also had an env var that forces linking it.. but I can't seem to get it to link to bzip2 at all, at least it looks to me like it doesn't.

```
=> openpgp-card-tools-0.10.0-r0: running pre_pkg hook: 001_runtime_deps...
   SONAME: libpcsclite.so.1 <-> pcsc-lite
   SONAME: libnettle.so.8 <-> nettle
   SONAME: libhogweed.so.6 <-> nettle
   SONAME: libgmp.so.10 <-> gmp
   SONAME: libunwind.so.1 <-> libunwind
   SONAME: libc.so <-> musl
=> openpgp-card-tools-0.10.0-r0: running pre_pkg hook: 002_cmd_provides...
```

any ideas?

Reply:
According to `cargo tree -f "{p} {f}" -i bzip2-sys`, it does not activate the `static` feature on bzip2-sys either:

```
bzip2-sys v0.1.11+1.0.8 
└── bzip2 v0.4.4 
    ├── buffered-reader v1.2.0 bzip2,compression-bzip2,compression-deflate,flate2
    │   └── sequoia-openpgp v1.16.1 bzip2,compression,compression-bzip2,compression-deflate,crypto-nettle,default,flate2,nettle
    │       ├── openpgp-card-sequoia v0.2.0 default
    │       │   └── openpgp-card-tools v0.9.5 (/home/jcgruenhage/dev/codeberg.org/openpgp-card/openpgp-card-tools) default
    │       └── openpgp-card-tools v0.9.5 (/home/jcgruenhage/dev/codeberg.org/openpgp-card/openpgp-card-tools) default
    └── sequoia-openpgp v1.16.1 bzip2,compression,compression-bzip2,compression-deflate,crypto-nettle,default,flate2,nettle (*)
```

Reply:
Ah, I probably need to rebase..

Reply:
yep, that was it.